hummingbird/ml/_topology.py

```diff
@@ -39,15 +39,23 @@
 else:
     DataFrame = None
 
+if pandas_installed():
```

Comment:
Something must be wrong with the rebase here.

Also i think there is some other problem with the prior merge... it seems that this [already was added...twice](https://github.com/microsoft/hummingbird/blob/master/hummingbird/ml/_topology.py#L37)

Reply:
oooh. Let me look into this.